packages/bits/src/lib/wizard-v2/wizard-horizontal/wizard-horizontal.component.html

```diff
@@ -13,17 +13,16 @@
             [attr.aria-label]="step.ariaLabel || null"
             [attr.aria-labelledby]="(!step.ariaLabel && step.ariaLabelledby) ? step.ariaLabelledby : null"
             [index]="i"
-            [state]="getStepState(i, step.state)"
+            [stepState]="getStepState(i, step.state)"
```

Comment:
What's the reason for the rename here? Maybe there's a reason that I'm not thinking of, but could we just keep calling it `state`?

Reply:
The reason of the rename is to avoid confusion since the step header component does not have its own state, it only stores the step's state.